readthedocs/subscriptions/tests/test_daily_email.py

```diff
@@ -151,7 +149,6 @@ def test_subscription_ended_days_ago(
         self.assertEqual(mock_send_email.call_count, 1)
         mock_send_email.assert_has_calls([
             mock.call(
-                request=mock.ANY,
```

Comment:
the request parameter isn't being used at all withing the send email function or task, I think we should just remove it.

Reply:
👍🏼 on removing it, then.